src/components/pages/account-page/components/group-tab/GroupTab.tsx

```diff
@@ -73,12 +81,25 @@ const GroupTab: FC = () => {
 
   const showRequests =
     data?.requests?.length !== 0 && user?.group?.role !== UserGroupRole.STUDENT;
+  data?.requests?.length !== 0 && user?.group?.role !== UserGroupRole.STUDENT;
 
   return (
-    <div className={styles['content']}>
-      <div className={styles['text-content']}>
-        <h4>Список групи {user.group.code}</h4>
-      </div>
+    <Box className={styles['content']}>
+      <Box className={styles['text-content']}>
+        <Typography variant="h4">Список групи {user.group.code}</Typography>
+        <IconButton
+          // TODO move inline styles when refactor
+          sx={{ height: 'fit-content', marginLeft: '10px' }}
+          shape={IconButtonShape.SQUARE}
+          color={IconButtonColor.SECONDARY}
+          icon={isSorted ? <ChevronDownIcon /> : <ChevronUpIcon />}
+          onClick={() => {
+            setIsSorted(!isSorted);
+            refetch();
```

Comment:
Why do we need a `refetch` here?

Reply:
Because otherwise student table will not update when sorting